### DISCUSSION TITLE
Playback Issues

### DISCUSSION THREAD

**Discussion by ghost** [kisslinux/repo#338] [Q&A]
Playback on Firefox and mpv are way too slow (can't even play 720p without dropping frames, but on Void Linux Musl does 1440p 60FPS no dropping) and I can't get the audio working (already installed the packages and built-in kernel support).

    $ lspci -k
    00:1f.2 Class 0101: 8086:1e00 ata_piix
    00:1f.0 Class 0601: 8086:1e47 lpc_ich
    00:01.0 Class 0604: 8086:0151 pcieport
    00:16.0 Class 0780: 8086:1e3a mei_me
    01:00.0 Class 0300: 10de:1c81 nouveau
    00:1b.0 Class 0403: 8086:1e20 snd_hda_intel
    00:1f.5 Class 0101: 8086:1e08 ata_piix
    00:19.0 Class 0200: 8086:1502 e1000e
    00:1e.0 Class 0604: 8086:244e
    00:1f.3 Class 0c05: 8086:1e22 i801_smbus
    00:00.0 Class 0600: 8086:0150 ivb_uncore
    00:1a.0 Class 0c03: 8086:1e2d ehci-pci
    00:16.3 Class 0700: 8086:1e3d serial
    00:1d.0 Class 0c03: 8086:1e26 ehci-pci
    01:00.1 Class 0403: 10de:0fb9 snd_hda_intel
    00:14.0 Class 0c03: 8086:1e31 xhci_hcd
    
    $ kiss l
    adwaita-icon-theme 41.0-1
    alsa-lib 1.2.5.1-1
    alsa-utils 1.2.5.1-1
    atk 2.36.0-1
    baseinit 1.0.3-1
    baselayout 1-8
    binutils 2.37-2
    bison 3.8.2-1
    bkeymaps 1.13-1
    busybox 1.34.0-6
    bzip2 1.0.8-1
    cairo 1.16.0-1
    cbindgen 0.20.0-1
    clang 12.0.1-1
    cmake 3.21.3-1
    curl 7.79.1-1
    dosfstools 4.2-1
    e2fsprogs 1.46.4-1
    expat 2.4.1-1
    ffmpeg 4.4-1
    file 5.40-1
    firefox 92.0-1
    firefox-privacy 1.0-4
    flex 2.6.4-4
    fontconfig 2.13.94-2
    foot 1.9.0-1
    freetype-harfbuzz 2.11.0+3.0.0-1
    gcc 11.2.0-1
    gdk-pixbuf 2.42.6-2
    git 2.33.0-1
    glib 2.70.0-1
    gnupg1 1.4.23-2
    grub 2.06-1
    gtk+3 3.24.30-1
    intel-vaapi-driver 2.4.1-1
    json-c 0.15-1
    kiss 5.5.28-1
    lame 3.100-2
    libass 0.15.2-1
    libdrm 2.4.107-1
    libelf 0.185-1
    libffi 3.4.2-2
    libinput 1.19.0-1
    libjpeg-turbo 2.1.1-1
    libogg 1.3.5-1
    libpciaccess 0.16-1
    libpng 1.6.37-2
    libseat 0.5.0-2
    libudev-zero 1.0.0-1
    libva 2.13.0-1
    libva-utils 2.12.0-1
    libvorbis 1.3.7-1
    libvpx 1.10.0-1
    libwebp 1.2.1-2
    libxkbcommon 1.3.1-1
    linux-headers 5.10.54-1
    llvm 12.0.1-1
    m4 1.4.19-1
    make 4.3-1
    man-pages 5.13-1
    mandoc 1.14.6-1
    mesa 21.2.2-1
    meson 0.59.1-1
    mpv 0.33.1-1
    musl 1.2.2-1
    nasm 2.15.05-1
    ncurses 6.2-5
    nodejs 16.10.0-1
    opendoas 6.8.1-2
    openssl 3.0.0-1
    opus 1.3.1-1
    pango 1.48.10-1
    pcre 8.45-1
    perl 5.34.0-3
    pfetch 0.6.0-1
    pigz 2.6-2
    pixman 0.40.0-1
    pkgconf 1.8.0-1
    python 3.9.7-1
    rust 1.55.0-1
    samurai 1.2-1
    sdhcp git-1
    sqlite 3.36.0-1
    sway-tiny c34a4ca6d234177416dc931efd3035f3b803315b-2
    ttf-croscore 1.31.0-2
    tzdata 2021b-1
    util-linux 2.37.2-1
    vim git-1
    wayland 1.19.0-1
    wayland-protocols 1.23-1
    wget 1.21.2-1
    wlroots 0.14.1-1
    x264 20190812-2245-4
    x265 3.5-1
    xkeyboard-config 2.33-1
    xvidcore 1.3.7-1
    xz 5.2.5-1
    zlib 1.2.11-3
    
    $ mpv
    ALSA lib pcm_direct.c:2133:(_snd_pcm_direct_new) unable to create IPC semaphore
    [ao/alsa] Playback open error: Function not implemented
    [ao] Failed to initialize audio driver 'alsa'
    Could not open/initialize audio device -> no sound.

Kernel [configuration](https://pastebin.com/F770XvRj).

**Comment by dylanaraps** [comment]
This sounds to me like a kernel issue though I cannot be sure. I cannot reproduce your issue here.
